Makefile

```diff
@@ -324,6 +324,22 @@ ifneq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \"
 	@echo 'WARN: Please use the recommended operator-sdk if you face any issue.'
 endif
 
+inject_kubeconfig:
```

Comment:
It's here just for test purpose or we are going to use it during development?

Reply:
Ideally it'll be used during development, provided I can get it to work. The idea is you would use che-machine-exec to log in the workspace terminal as the devworkspace-controller serviceaccount, allowing you to run/debug "locally" and have it run in the cluster like a normal controller.